src/components/daydream-controls.js

```diff
@@ -83,7 +81,14 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var handedness = this.data.hand === '' ? ['left', 'right'] : [this.data.hand];
+    handedness = handedness.filter(function (hand) {
+      return this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: hand});
+    }.bind(this));
+    var isPresent = handedness.length > 0;
+    if (handedness.length > 0) {
+      this.el.setAttribute('daydream-controls', { hand: handedness[0] });
+    }
```

Comment:
(this will fail if there is one of each hand, but I don't think that is a real-world use case for Daydream at the moment.)

this will fail to ever allow a match if the daydream controller is ever missing hand property, not sure if that ever happens as real-world case at system level, but might happen in browser with broken support; my suggestion / request is to make it such that if no hand filter is requested, no hand filter is applied at all, just take what's there if anything (rather than trying with both hand filters separately and forcing one to apply)